htdocs/GetCSS.php

```diff
@@ -28,9 +28,11 @@
 // Ensures the user is logged in, and parses the config file.
 require_once "NDB_Client.class.inc";
 $client = new NDB_Client();
-
-if ($client->initialize(null) == false) {
-    return false;
+try {
+    $anonymous = ($client->initialize("../project/config.xml") === false);
+} catch (Exception $e) {
+    echo "Exception caught: client could not initialize... ", $e->getMessage(), "\n";
+    $anonymous = false;
```

Comment:
@driusan should this be `$anonymous = true;` ?

Reply:
this is not outdated

Reply:
`$anonymous = true;` makes more sense to me. If there was an exception initializing the client, we should assume we don't know who the user is.